### ISSUE TITLE
[a11y] Slash - Alert - Close button should have better aria-label and be changed

### ISSUE THREAD

**Issue by MartinWeb** [AxaFrance/design-system#982]
The close button on the alert component has an aria-label "close".
Maybe this default value should be changed, but projects should be able to change it to add better explaning to users.

Add a props to set this aria-label.

**Comment by LordGregDavid** [comment]
Je la prends
